webpack/ForemanTasks/Components/TasksTable/Components/ConfirmModal/ConfirmModal.js

```diff
@@ -0,0 +1,66 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { startCase } from 'lodash';
+import { sprintf, translate as __ } from 'foremanReact/common/I18n';
+import { Button } from 'patternfly-react';
+import ForemanModal from 'foremanReact/components/ForemanModal';
+
+export const ConfirmModal = ({
+  actionText,
+  actionState,
+  actionType,
+  selectedRowsLen,
+  id,
+  parentTaskID,
+  url,
+  uriQuery,
+  setModalClosed,
+  ...props
+}) => (
+  <ForemanModal
+    title={sprintf(__('%s Selected Tasks'), startCase(actionText))}
+    id={id}
+  >
+    {sprintf(
+      __(
+        `This will %(action)s %(number)s task(s), putting them in the %(state)s state. Are you sure?`
+      ),
+      { action: actionText, number: selectedRowsLen, state: actionState }
+    )}
+    <ForemanModal.Footer>
+      <Button onClick={setModalClosed}>{__('No')}</Button>
+      <Button
+        className="confirm-button"
+        bsStyle="primary"
+        onClick={() => {
+          props[actionType]({ url, query: uriQuery, parentTaskID });
```

Comment:
nit: can you rename `uriQuery` to `query` in the props that are passed from the parent?

Reply:
Sure, renamed